src/Flysystem/FlysystemAssetStore.php

```diff
@@ -417,21 +418,23 @@ protected function moveBetweenFilesystems($fileID, Filesystem $from, Filesystem
 
     public function grant($filename, $hash)
```

Comment:
For a like a change to this public API will be needed to avoid the `Controller::curr()` reference. I don't think it blocks this PR but I wonder if we should do that before beta1?